src/main/scala/com/databricks/spark/avro/AvroRelation.scala

```diff
@@ -117,6 +118,7 @@ private[avro] class AvroRelation(
    * contain `requiredColumns`
    */
   override def buildScan(requiredColumns: Array[String], inputs: Array[FileStatus]): RDD[Row] = {
+    val withalias = withAliasFields
```

Comment:
while accessing directly class variable 'withAliasFields' inside the map 'superSchema.getFields.map' which calls hashcode() function. In spark 1.4.0, getting Class not found exception for guava class 'Objects' in hashcode() function.